go.mod

```diff
@@ -1,10 +1,14 @@
 module github.com/openfaas/mqtt-connector
 
-go 1.15
+go 1.18
 
 require (
 	github.com/eclipse/paho.mqtt.golang v1.2.0
 	github.com/openfaas/connector-sdk v0.6.1
 	github.com/openfaas/faas-provider v0.17.1
+)
+
+require (
+	github.com/pkg/errors v0.9.1 // indirect
```

Comment:
I'd like to remove all usage of errors.Wrap and change it over for this, if we have that code in our own files in this project.

```golang
fmt.Errorf("some additional context, error: %w", err)
```

Could you do that too?